src/benchmarks/gc/src/exec/run_single_test.py

```diff
@@ -306,27 +306,29 @@ def _get_perfview_start_or_stop_cmd(
 
 _DEFAULT_MAX_TRACE_SIZE_GB = 1
 
+_collect_options = {
+    CollectKind.gc: ["-GCCollectOnly"],
+    CollectKind.verbose: ["-GCCollectOnly", "-ClrEventLevel:Verbose"],
+    # Need default kernel events to get the process name
+    CollectKind.cpu_samples: [
+        "-OnlyProviders:ClrPrivate:1:5,Clr:1:5",
+        "-ClrEvents:GC+Stack",
+        "-ClrEventLevel:Verbose",
+        "-KernelEvents:Default",
+    ],
+    CollectKind.thread_times: [
+        "-OnlyProviders:ClrPrivate:1:5,Clr:1:5",
+        "-ClrEvents:GC+Stack",
+        "-ClrEventLevel:Verbose",
```

Comment:
>"-ClrEventLevel:Verbose", [](start = 8, length = 25)

do you actually need this? we are already saying 5 for the level in OnlyProviders

Reply:
I forgot to remove that line. I'll change it.